test/e2e/specs/editor/various/post-visibility.spec.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.describe( 'Post visibility', () => {
+	const wp = '';
+	test.afterEach( async ( { pageUtils } ) => {
+		await pageUtils.setBrowserViewport( 'large' );
+	} );
```

Comment:
Since we're recreating the `page` instance for every test, I think we don't need to reset it after each test. Let's try removing it?

```suggestion
```